docs/nzxt-kraken-x3-devices.md

```diff
@@ -0,0 +1,99 @@
+# NZXT X3 devices
+
+## NZXT Kraken X53, X63, X73
+
+The Kraken X53, X63 and X73 compose the fourth generation of liquid coolers by NZXT.  These devices are manufactured by Asetek and house seventh generation Asetek pumps, plus secondary PCBs specially designed by NZXT for enhanced control and lighting.
+
+They incorporate customizable pump speed control, a liquid temperature probe in the block and addressable RGB lighting.  The coolers are powered directly by the power supply unit.
+
+All configuration is done through USB, and persists as long as the device still gets power, even if the system has gone to Soft Off (S5) state.  The cooler also reports pump speed and liquid temperature via USB; pump speed can also be sent to the motherboard (or other device) via the sense pin of a standard fan connector.
```

Comment:
> configuration persists as long as the device is powered on

Yes.

> the controller is powered on by the USB bus, so settings can persist even in Soft Off (S5) state (if there's 5V Stand-by power, which depends on the motherboard's ErP settings)

It needs a SATA connection for power, I guess? But it does keep the configuration until you completely cut the power of the pc.

> pump speed is reported to the motherboard (or other device) via the sense pin of a standard fan connector

There is a CPU fan in my BIOS settings with a reported RPM value.

> Additionally, does that connector (standard fan connector from pump to motherboard) have any other pins populated besides sense and ground? This would determinate if it's possible to control the pump from BIOS (which wasn't the case for the X[4-7]2 coolers).

I didn't inspect the pins yet but changing the fan curves in BIOS didn't change anything if I remember correctly.


I can't speak for all X[5-6] devices, maybe we should ask this in issue #78?

Reply:
> It needs a SATA connection for power, I guess? But it does keep the configuration until you completely cut the power of the pc.

With the third gen. SATA power is only used for the mechanical (pump/fans) or high-current electrical (LEDs) things.  The embedded controller is completely powered from USB 5V (which in turn can be powered by +5V Standby).

From what you describe, the same is true with forth gen., as a standard SATA connector doesn't supply +5V Standby and the cooler would reset even in Soft Off (AFAIK even suspend-to-memory).

> There is a CPU fan in my BIOS settings with a reported RPM value.
> I didn't inspect the pins yet but changing the fan curves in BIOS didn't change anything if I remember correctly.

Again similar to third gen., which makes sense.

> I can't speak for all X[5-6] devices, maybe we should ask this in issue #78?

Yeah, we probably should, even though I don't expect there to be any differences, given that in all other control aspects the three SKUs are equal, indistinguishable even.

But I vaguely remember some old family of coolers where the smallest SKU had an extra physical limitation... I think you couldn't control the LED, even though on the USB/protocol sides it was identical to its larger brothers.